### PR TITLE
Fix rabbitmq and redis images to use our prod-like versions.

### PR DESCRIPTION
**Pull request by EnTeQuAk**
We're actually using rabbitmq 3.5.3-1 and redis 2.8.23 in production but I think this comes pretty close.

@jasonthomas what version do we use for memcached? I suppose something like either 1.4.5 or 1.4.2?

### CODE REVIEW

**Comment by jasonthomas** [comment]
We are using 1.4.24 in our envs today.

File: docker-compose.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ nginx:
     - web:web
 
 memcached:
-  image: memcached
+  image: memcached:1.4.24
```
Comment (muffinresearch) [inline docker-compose.yml]: If you pin versions to this degree it does mean we don't get updates without manual intervention. E.g. if a security patch came out in `1.4.25` or `1.5` we won't get it.

Comment (EnTeQuAk) [inline docker-compose.yml]: I can tag it to 1.4 but I assume 1.5 for memcached would be a major release.

**Comment by EnTeQuAk** [comment]
Merged in 441ed5f

**Pull request closed without merging**